rasa/cli/run.py

```diff
@@ -30,6 +30,8 @@ def add_subparser(
     )
     run_parser.set_defaults(func=run)
 
+    # NOTE: subparsers here need special handling in
```

Comment:
I think we can remove the note here. Instead I would add a comment to `parse_last_positional_argument_as_model_path`.

Reply:
I changed comments per your recommendation. Hopefully I got what you were looking for.